### PR TITLE
Added learn more URL to deposits page once

### LINKED ISSUE #5370: Payments > Overview improvement one
Filing issues for some improvements from this post: pbUcTB-1Rx-p2

The "Learn more" link under Payments > Overview goes away after the merchant’s first deposit. However, a very common question merchants have is specifically about deposit timing! We should consider leaving the link in some form even after the initial deposit / waiting period.

image

After the 7 days, the box changes to look like the below screenshot. However, the “Change” link is not deep-linked to the Deposits section of the Settings page.

image

Proposed text after 7 days: Deposits set to XYZ. [Change this](#) or [learn more](#).

"Change this" would be linked specifically to the Payments > Settings > Deposits header. Can we do this?

### PR DESCRIPTION
**Pull request by zmaglica**
Fixes #5370 

#### Changes proposed in this Pull Request
This PR adds "learn more" URL next to "Change" url on deposits screen.

#### Testing instructions

Make sure that you go to the linked issue and pay attention what needs to be changed and make sure that all changes are visible. Make sure that you test this with older account since you won't see the change if your account is new one since deposits are locked for new accounts.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: client/deposits/utils/index.js [hunk 1]
```diff
@@ -172,6 +176,16 @@ export const getDepositScheduleDescriptor = ( {
...
 					/>
 				),
+				learn_more_href: (
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						href={ learnMoreHref }
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						{ __( 'learn more' ) }
```
Comment (RadoslavGeorgiev) [inline client/deposits/utils/index.js]: The `woocommerce-payment` text domain is missing here.

File: client/deposits/utils/index.js [hunk 2]
```diff
@@ -158,7 +159,10 @@ export const getDepositScheduleDescriptor = ( {
 		return createInterpolateElement(
 			sprintf(
 				/** translators: %s - deposit schedule, <a> - Settings page URL */
-				__( '%s <a>Change</a>', 'woocommerce-payments' ),
+				__(
+					'%s <a>Change this</a> or <learn_more_href/>',
```
Comment (RadoslavGeorgiev) [inline client/deposits/utils/index.js]: Let's use the punctuation, suggested in the issue, in this case add the missing dot at the end:

> Deposits set to XYZ. [Change this](#) or [learn more](#).

File: client/deposits/utils/index.js [hunk 3]
```diff
@@ -129,9 +129,10 @@ export const getDepositScheduleDescriptor = ( {
 		);
 	}
 
+	const learnMoreHref =
```
Comment (RadoslavGeorgiev) [inline client/deposits/utils/index.js]: Considering this is a constant for the whole method, should we move it next to `isCustomDepositSchedulesEnabled` and `hasCompletedWaitingPeriod` in the beginning of the code block?

**Review by RadoslavGeorgiev — CHANGES_REQUESTED** [review]
Great work! 👏 

I have a couple of nitpicks, otherwise the PR looks and works great.

---

Keep it up with the granular commits! 🙇

**Review by naman03malhotra — COMMENTED** [review]
One of the expectations of the Issue is to deep link "Change link" text to `Payments > Settings > Deposits header`, has this change been accounted for in this PR?

File: client/deposits/utils/index.js (hunk 2 above)
Comment (zmaglica) [reply client/deposits/utils/index.js]: Fixed.

File: client/deposits/utils/index.js (hunk 3 above)
Comment (zmaglica) [reply client/deposits/utils/index.js]: Fixed. Const is at the beginning of the code block

**Comment by zmaglica** [comment]
> One of the expectations of the Issue is to deep link "Change link" text to `Payments > Settings > Deposits header`, has this change been accounted for in this PR?

Thank you for pointing this out. I added an anchor link, and it will be anchored to deposits section in settings page.

**Review by RadoslavGeorgiev — APPROVED** [review]
Looks good, thank you for the fixes! :shipit:

**Pull request merged**